src/pages/waiting-room-to-car/cat-be/components/vehicle-checks-modal/vehicle-checks-modal.cat-be.page.ts

```diff
@@ -68,6 +76,13 @@ export class VehicleChecksCatBEModal {
         select(getVehicleChecksCatBe),
         select(getSelectedShowMeQuestions),
       ),
+      tellMeQuestions$: this.store$.pipe(
+        select(getTests),
```

Comment:
Would extract 
```
const currentTest$ = this.store$.pipe(
    select(getTests),
    select(getCurrentTest),
);
```

And reuse it when creating these observables

Reply:
ahh right ok, do you mean something like this?:-

```
  ngOnInit(): void {
    const currentTest$ = this.store$.pipe(
      select(getTests),
      select(getCurrentTest),
    )
    this.pageState = {
      candidateName$: currentTest$.pipe(
        select(getJournalData),
        select(getCandidate),
        select(getUntitledCandidateName),
      ),
      showMeQuestions$: currentTest$.pipe(
        select(getTests),
        select(getCurrentTest),
        select(getTestData),
        select(getVehicleChecksCatBe),
        select(getSelectedShowMeQuestions),
      ),
      tellMeQuestions$: currentTest$.pipe(
        select(getTests),
        select(getCurrentTest),
        select(getTestData),
        select(getVehicleChecksCatBe),
        select(getSelectedTellMeQuestions),
      )
    };
  }
```

Reply:
Yeah that's right